codechain/__init__.py

```diff
@@ -0,0 +1,20 @@
+import sys
+
+from .crypto import *
+from .primitives import *
+
+# -------
+# Pythons
+# -------
+
+# Syntax sugar.
+_ver = sys.version_info
+
+#: Python 2.x?
+is_py2 = _ver[0] == 2
+
+#: Python 3.x?
+is_py3 = _ver[0] == 3
+
+if is_py2:
```

Comment:
```
if sys.version_info < (3, 6):
```

Could you add `python_requires=">=3.6"` in arguments of `setup` in `setup.py` too?

Reply:
Done.